CHANGELOG.md

```diff
@@ -5,6 +5,62 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.1.0] 2022-01-21
+
+The 3.1.0 release is a minor release for the plugin that does not introduce any breaking changes coming from the 3.0 branch. This version's primary focus is adding support for WordPress decoupled architectures and a revamped settings page. We have also worked hard on refining our code, testing, and delivery process.
```

Comment:
I wouldn't call this release a minor release. Also not sure if the plural "decoupled architectures" is correct.

I would probably change the text like so:

This version's primary focus is adding support for WordPress decoupled architecture and a revamped settings page. We have also worked hard on refining our code, testing, and delivery process. This release does not introduce any breaking changes when upgrading from the 3.0 branch.

Reply:
It is a minor release as per the definition we're using: https://semver.org/